src/js/step.js

```diff
@@ -302,16 +257,16 @@ export class Step extends Evented {
       delete this.el;
     }
 
-    if (this.popper) {
-      this.popper.destroy();
+    if (this.tooltipElem) {
+      this.tooltipElem.destroy();
+      this.tooltipElem = null;
```

Comment:
I see where you're coming from, but I think that might actually be a reason _not_ to place it outside of the `if` statement: `this.tooltipElem` should never be undefined, false, 0, etc -- and if something ever failed because of that, we'd probably benefit by having it do so and getting to the root of the cause.

Reply:
If that's the case, then should this be a try/catch then?

Reply:
I guess what I'm saying is we should be able to have absolute confidence and control over what `this.tooltipElem` -- so we shouldn't need to get _more_ advance with the checking either. 

If something fails in the future because someone has `this.tooltipElem` set to `0`, that's something we'd want to address then and there to discover how that could happen.